tst/EnergyPlus/unit/ScheduleManager.unit.cc

```diff
@@ -1741,11 +1746,15 @@ TEST_F(EnergyPlusFixture, ScheduleCompact_MissingDayTypes)
     // Test for #11054
     std::string const idf_objects = delimited_string({
         "ScheduleTypeLimits,",
-        "  Any Number;              !- Name",
+        "  Fraction,                !- Name",
+        "  0,                       !- Lower Limit Value",
+        "  1,                       !- Upper Limit Value",
+        "  Continuous,              !- Numeric Type",
+        "  percent;                 !- Unit Type",
 
         "Schedule:Compact,",
         "  WindowVentSched,         !- Name",
-        "  Any Number,              !- Schedule Type Limits Name",
+        "  Fraction,                !- Schedule Type Limits Name",
```

Comment:
Mod the previous test to add that ScheduleTypeLimits validation which makes it crash before fix in setMinMaxVals like the new defect file does